### DISCUSSION TITLE
date format

### DISCUSSION THREAD

**Discussion by Alllfff** [robbrad/UKBinCollectionData#636] [Ideas]
### Home Assistant Version

2024.3

### Installation Method

Using HACS

### Name of Council (if relevant)

West Lothian

### Issue Information

How can i change the date format?

this is how entity is shown
![Screen Shot 2024-03-14 at 11 42 00](https://github.com/robbrad/UKBinCollectionData/assets/39865207/9af10e1f-1248-4863-aa12-06bae85b2b64)
this is what is says on the council website
![Screen Shot 2024-03-14 at 11 43 14](https://github.com/robbrad/UKBinCollectionData/assets/39865207/0bac59d4-6c3c-41a9-85a6-fef9bb03db3b)
I would like it to either say what is on the council website (ie April 5th 2024) or decide on DD/MM/YYYY format.
Thank you

### Verification

- [X] I searched for similar issues at https://github.com/robbrad/UKBinCollectionData/issues?q=is:issue and found no duplicates
- [X] If trying to add a specific council, I've checked it is listed as working at https://robbrad.github.io/UKBinCollectionData/3.11/
- [X] I have provided a detailed explanation of the issue as well as steps to replicate the issue
- [X] I understand that this project is run by volunteer contributors therefore completion of this issue cannot be guaranteed

**Comment by dp247** [comment]
Converted to a discussion as its not a bug, but might be worth having as an enhancement

**Comment by swwebb1** [comment]
@Alllfff - Im trying to set this up and having some struggles. Without doxxing myself - also looking to set it up for WLC……

Can I PM you with some queries as to how you got this running?

**Reply by Alllfff** [reply]
Of course, go for it

**Reply by swwebb1** [reply]
Huh, not often on SO & realised they don’t have a DM function as far as I can see….

I think I have got the local selenium setup enabled for my HA instance. Once it was set up, how did you execute a scrape data into a file? Have you integrated with this? https://github.com/mampfes/hacs_waste_collection_schedule/tree/master

**Reply by Alllfff** [reply]
Sorry I totally missed your reply. 
I do not use https://github.com/mampfes/hacs_waste_collection_schedule/tree/master or scrape data into a file (as far as I know? My knowledge of HA is pretty basic, please understand)

I only have 4 sensors on my dashboard that look like this:
![Screen Shot 2024-04-05 at 10 29 19](https://github.com/robbrad/UKBinCollectionData/assets/39865207/8752e29e-a4a2-40b2-a4d7-311534185e67)
no schedules or automations based on this (yet)

Sorry I could not help you more...

**Comment by jamescorday** [comment]
```
type: vertical-stack
cards:
  - type: custom:mushroom-template-card
    entity: sensor.test_lothian_blue_bin
    icon: mdi:trash-can
    icon_color: blue
    primary: '{{ state_attr(''sensor.test_lothian_blue_bin'',''friendly_name'') }}'
    secondary: '{{ state_attr(''sensor.test_lothian_blue_bin'',''next_collection'').strftime("%d-%m-%Y") }}'
  - type: custom:mushroom-template-card
    entity: sensor.test_lothian_brown_bin_now_for_garden_and_food_waste
    icon: mdi:trash-can
    icon_color: brown
    primary: '{{ state_attr(''sensor.test_lothian_brown_bin_now_for_garden_and_food_waste'',''friendly_name'') }}'
    secondary: '{{ state_attr(''sensor.test_lothian_brown_bin_now_for_garden_and_food_waste'',''next_collection'').strftime("%d-%m-%Y") }}'
  - type: custom:mushroom-template-card
    entity: sensor.test_lothian_green_bin
    icon: mdi:trash-can
    icon_color: green
    primary: '{{ state_attr(''sensor.test_lothian_green_bin'',''friendly_name'') }}'
    secondary: '{{ state_attr(''sensor.test_lothian_green_bin'',''next_collection'').strftime("%d-%m-%Y") }}'
  - type: custom:mushroom-template-card
    entity: sensor.test_lothian_grey_bin
    icon: mdi:trash-can
    icon_color: grey
    primary: '{{ state_attr(''sensor.test_lothian_grey_bin'',''friendly_name'') }}'
    secondary: '{{ state_attr(''sensor.test_lothian_grey_bin'',''next_collection'').strftime("%d-%m-%Y") }}'

**Reply by jamescorday** [reply]
Hopefully this helps, if you never manged to get your lovelace code in order :)

![Screenshot 2024-05-01 at 17 44 58](https://github.com/robbrad/UKBinCollectionData/assets/121184812/2c798352-2b2e-420c-9072-5db1311a23d5)
